scripts/updatecli/updatecli_config.yml

```diff
@@ -0,0 +1,239 @@
+      directory: '/tmp/updatecli/search'
+  searchEnterpriseRepo:
+    kind: git
+    spec:
+      url: git@github.com:Alfresco/alfresco-elasticsearch-connector.git
+      branch: master
+      directory: '/tmp/updatecli/search_enterprise'
+  transformRepo:
+    kind: git
+    spec:
+      url: git@github.com:Alfresco/alfresco-transform-core.git
+      branch: master
+      directory: '/tmp/updatecli/transform'
+  trouterRepo:
+    kind: git
+    spec:
+      url: git@github.com:Alfresco/alfresco-transform-service.git
+      branch: master
+      directory: '/tmp/updatecli/trouter'
+
+sources:
+  acs:
+    name: ACS {{ .acs.version }}.x
+    kind: gittag
+    scmid: acsRepo
+    spec:
+      versionFilter:
+        kind: regex
+        pattern: '{{ .acs.version }}.(\d+){{ .version_pattern }}'
+  adw:
+    name: ADW {{ .adw.version }}.x
+    kind: gittag
+    scmid: adwRepo
+    spec:
+      versionFilter:
+        kind: regex
+        pattern: '^{{ .adw.version }}.(\d+){{ .version_pattern }}'
+  aosAmp:
+    name: AOS AMP
+    kind: gittag
+    scmid: aosAmpRepo
+    spec:
+      versionFilter:
+        kind: regex
+        pattern: '{{ .aos.version }}.(\d+){{ .version_pattern }}'
+  googleDriveAmp:
+    name: Google Drive AMP
+    kind: gittag
+    scmid: googleDriveAmpRepo
+    spec:
+      versionFilter:
+        kind: regex
+        pattern: '{{ .googleDrive.version }}.(\d+){{ .version_pattern }}'
+  apiExplorer:
+    name: Api explorer {{ .apiExplorer.version }}.x
+    kind: gittag
+    scmid: apiExplorerRepo
+    spec:
+      versionFilter:
+        kind: regex
+        pattern: '{{ .apiExplorer.version }}.(\d+){{ .version_pattern }}'
+  dsync:
+    name: Desktop Sync {{ .dsync.version }}
+    kind: gittag
+    scmid: dsyncRepo
+    spec:
+      versionFilter:
+        kind: regex
+        pattern: '{{ .dsync.version }}.(\d+){{ .version_pattern }}'
+  search:
+    name: InsightEngine {{ .search.version }}.x
+    kind: gittag
+    scmid: searchRepo
+    spec:
+      versionFilter:
+        kind: regex
+        pattern: '{{ .search.version }}.(\d+){{ .version_pattern }}'
+  searchEnterprise:
+    name: Search Enterprise {{ .searchEnterprise.version }}.x
+    kind: gittag
+    scmid: searchEnterpriseRepo
+    spec:
+      versionFilter:
+        kind: regex
+        pattern: '{{ .searchEnterprise.version }}.(\d+){{ .version_pattern }}'
+  transform:
+    name: Transform Core {{ .transform.version }}.x
+    kind: gittag
+    scmid: transformRepo
+    spec:
+      versionFilter:
+        kind: regex
+        pattern: '{{ .transform.version }}.(\d+){{ .version_pattern }}'
+  trouter:
+    name: ATS Transform Service {{ .trouter.version }}.x
+    kind: gittag
+    scmid: trouterRepo
+    spec:
+      versionFilter:
+        kind: regex
+        pattern: '{{ .trouter.version }}.(\d+){{ .version_pattern }}'
+
+
+targets:
+  acs:
+    name: Bump acs
+    kind: yaml
+    sourceid: acs
+    spec:
+      key: acs.version
+      file: group_vars/all.yml
```

Comment:
Can't we use a variable from the yaml files to set the targets? (so we can support updating extra-vars files?)